README.md

```diff
@@ -90,7 +90,8 @@ Copy `packages/vsf-storyblok-extension` to `vue-storefront-api/src/api/extension
 ...
 "storyblok": {
   "previewToken": "__API_KEY_HERE__",
-  "hookSecret": "__SECRET_CHANGE_ME__"
+  "hookSecret": "__SECRET_CHANGE_ME__",
+  "invalidate": "http://localhost:3000/invalidate?tag=*&key=aeSu7aip"
```

Comment:
Should we have `tag=storyblok` instead of `tag=*`?
`tag=*` would clear all the product and category caches as well, right?

Reply:
You're right, thanks!